esp-config/src/generate/value.rs

```diff
@@ -118,3 +182,22 @@ impl From<String> for Value {
         Value::String(value)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn deserialization_number() {
```

Comment:
Can we add some tests for round tripping with some large numbers? I just want to ensure that the i128 is still respected with the string parsing